tests/dummy/app/pods/components/user-menu/template.hbs

```diff
@@ -0,0 +1,46 @@
+{{frost-button
+  class=(concat css '-user-button')
+  hook=(concat hookPrefix '-userButton')
+  hookQualifiers=hookQualifiers
+  icon='user'
+  pack='frost-nav'
+  text=username
+}}
+
+{{#frost-popover
+  class='frost-navigation-user-menu'
```

Comment:
Actually, maybe in another PR. I'm gonna try this one with just the style updates and the example user-menu this time, since I'm not sure the best way to allow consumers to define their own additional items in the user menu dropdown.